drivers/dma/dma_stm32.c

```diff
@@ -117,7 +117,7 @@ static void dma_stm32_irq_handler(void *arg)
 	}
 }
 
-static u32_t dma_stm32_width_config(struct dma_config *config,
+static int dma_stm32_width_config(struct dma_config *config,
```

Comment:
This change is not documented

Reply:
like other functions in this file, `dma_stm32_width_config` cannot return `-EINVAL` as `u32_t`
return value must be typed as **int**.